org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileSearchQuery.java

```diff
@@ -261,6 +271,18 @@ public String getSearchString() {
 	}
 
 	public String getResultLabel(int nMatches) {
+		String resultLabel = getResultLabelWithOutLimitString(nMatches);
+
+		int maxResultAllowed = SearchPreferencePage.getMaxResults();
```

Comment:
reusue maxSearchResults

Reply:
I'm not sure I can, it's in a different class that's static. I don't think I have a reference to the collector here. I could change the collector to be an instance variable of this class, not sure if that's better or worst than just asking for the value from preferences again or not. If performance is a concern, I could make the maxSearchResults an instance variable, seems less intrusive than making the collector one.

Reply:
my idea was that i would like to see the value that was actually used.

Reply:
okay, amended, the msg now shows the limit you've set.